include/nil/blueprint/parser.hpp

```diff
@@ -1234,9 +1311,8 @@ namespace nil {
                                     } else if (ret_val->getType()->isVectorTy()) {
                                         std::vector<var> res = extracted_frame.vectors[ret_val];
                                         for (var x : res) {
-                                            std::cout << var_value(assignments[currProverIdx], x).data << " ";
+                                            std::cout << var_value(assignments[currProverIdx], x).data << "\n";
```

Comment:
`std::endl` is preferred for cout, it flushes stdout buffer.

Reply:
ok